### DISCUSSION TITLE
Subscribe button are bugged (thinks that i subscribed to issue when i'm not)

### DISCUSSION THREAD

**Discussion by soredake** [community/community#86703] [Mobile]
### Select Topic Area

Bug

### Body

Subscribe button is bugged (thinks that I subscribed to issue when I'm not), demonstration of the bug in WSL repository:

https://github.com/community/community/assets/5204968/5988dfdc-4100-4b45-acea-c264ff067219

**Comment by elisealix22** [comment]
Thank you so much for the video, @soredake ! That is extremely helpful in debugging. I noted this behavior in an internal issue and we will definitely look into it. Please let us know if you have any other feedback as well.
